setup.py

```diff
@@ -24,11 +24,31 @@
 # comment this out to disable multi threaded builds.
 import mp_compile
 
+
+def get_version():
+    version_file = "src/PIL/_version.py"
+    with open(version_file, "r") as f:
+        exec(compile(f.read(), version_file, "exec"))
+    return locals()["__version__"]
+
+
+NAME = "Pillow"
+PILLOW_VERSION = get_version()
+FREETYPE_ROOT = None
+IMAGEQUANT_ROOT = None
+JPEG2K_ROOT = None
+JPEG_ROOT = None
+LCMS_ROOT = None
+TIFF_ROOT = None
+ZLIB_ROOT = None
+
+
 if sys.platform == "win32" and sys.version_info >= (3, 9):
     warnings.warn(
-        "Pillow does not yet support Python {}.{} and does not yet provide "
-        "prebuilt Windows binaries. We do not recommend building from "
-        "source on Windows.".format(sys.version_info.major, sys.version_info.minor),
+        "Pillow {} does not yet support Python {}.{} and does not yet provide prebuilt"
```

Comment:
```suggestion
        "Pillow {} does not support Python {}.{} and does not provide prebuilt"
```
If we're talking about a specific Pillow version, then I don't think 'yet' is true anymore - Pillow 6.2.0 will never support Python 3.8, for example.

Reply:
Good point, thanks!